themes/default/content/docs/intro/pulumi-service/_index.md

```diff
@@ -12,15 +12,15 @@ aliases:
 - /docs/intro/console/
 ---
 
-The [Pulumi Service](https://app.pulumi.com) web application automatically manages deployment state and enables collaboration between developers and operators. The Pulumi CLI automatically uses it unless you [explicitly opt out]({{< relref "/docs/intro/concepts/state" >}}).
+The [Pulumi Service]({{< relref "/product/pulumi-service" >}}) is a free, managed service that handles infrastructure state and secrets and provides visibility into your cloud resources and their history. The Pulumi CLI automatically uses it unless you [explicitly opt out]({{< relref "/docs/intro/concepts/state" >}}). 
```

Comment:
i think its an interesting decision here to link from the docs to the marketing page instead of to the product itself. do you get alignment from folks on this change? i also think the `free` mention is a bit misleading. happy to approve though if you already have support from others.